Appraisals

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+appraise 'rubocop-0.49.1' do
+  gem 'gitlab', '<= 4.14.0'
```

Comment:
gitlab 4.14.1 and above requires ruby 2.5 or above  (NARKOZ/gitlab@421d37d3e49ccfd78b842b5155aed2562b0f749c)